### PR TITLE
Added metadata for examples/models/toolbars2.py

### PR DESCRIPTION
**Pull request by fllagami**
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [x] issues: fixes #xxxx
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

**Comment by fllagami** [comment]
Related to issue #11765

File: examples/models/toolbars2.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
+''' This example shows multiple ways to place the toolbar with respect to a grid of up to four plots.
```
Comment (bryevdv) [inline examples/models/toolbars2.py]: This example happens to have four plots, but there is nothing special about that number, these locations will work with any grid size.
```suggestion
''' This example shows multiple ways to place the toolbar with respect to a grid of plots.
```

File: examples/models/toolbars2.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+.. bokeh-example-metadata::
+    :apis: bokeh.layouts.column, bokeh.layouts.gridplot, bokeh.layouts.row, bokeh.plotting.figure, bokeh.plotting.show
+    :refs: :ref:`ug_interaction_tools_toolbar`
+    :keywords: tools, toolbar, mkplot, mkgrid, layout
```
Comment (bryevdv) [inline examples/models/toolbars2.py]: `mkgrid` and `mkplot` are helpers that exist only in this example, they are not relevant to anything else in a wider context
```suggestion
    :keywords: tools, toolbar, layout
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks for your patience @fllagami I've been sick recently. Just a couple of comments then this is good to merge

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
The documentation job failing is unrelated and will go away when this is merged (i.e. it is fixed on the main dev branch) thanks @fllagami

**Pull request merged**